<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">


<head>
<link href="../../styles/style2.css" rel="stylesheet" type="text/css" />
<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
<title>Figure 4. Esrrb regulated mRNA/miR feed-forward network motif.</title>
</head>

<body style="width: 1260px; height: 1740px;"/>

 <div class="image_popup" style="width: 1200px; height: 1737px;">
<a href="../figure5/figure5.png" target="_blank"><img alt="Figure 4" src="figure4.png"/></a><br>
<div style="width: 1197px; height: 118px">
	<strong>Figure 4. Co-expression map for gene responses to depletion of Nanog or Esrrb <br />
	</strong>
	(A) Co-expression network based on time-series data collected after Nanog or Esrrb downregulation. Nodes in the network are colour-coded according to the identified co-expression clusters; large nodes in red mark pluripotency genes, while large nodes in dark blue mark differentiation genes (gene names from each specific cluster are shown in Table S4). Boxes next to each co-expression cluster show average expression profiles across all genes in this cluster (line plots) and the average TF binding profiles (histograms) for this cluster. The histogram compares the fraction of known ChIP targets for a given TF in a cluster versus that expected based on entire genome. Three disconnected network domains correspond to genes with changing expression in both Nanog and Esrrb datasets (clusters 1-3), genes with changing expression only in the Esrrb dataset (clusters 4-6) and genes with changing expression only in the Nanog dataset (clusters 7-8). (B) Pair-wise comparisons of genes from different germ cell lineages. Colors represent mRNA expression levels in the two time courses (left side of the color bar). Orange or dark blue in LOF and GOF represent that loss or gain of function of the first gene in the pair activates or represses the second gene in the pair. Grey for LOF or GOF columns means NA (non assay) (middle region of the color bar). Finally in the ChEA column, dark orange or grey indicate whether the first gene-product of the pair binds to the promoter of the second gene of the pair according to the ChEA (Lachmann et al., 2010) database (right side of the color bar). For example the Esrrb-Klf4 pair presents the color code (down-down, left side of the color bar), which means that in both Esrrb and Nanog depletion time-courses, Esrrb and Klf4 are downregulated. The bright orange color in the LOF studies means that Esrrb activates Klf4 and the grey color (middle region of the color bar) in the GOF studies means that there are no available data from GOF studies. The grey color in the ChEA column means that Esrrb is not a target of Klf4 according to the studies collected in the ChEA (Lachmann et al., 2010) database (right side of the color bar). (C) Experimental validation of gene responses predicted from (Figure 4B). Expression levels of Esrrb, Nanog, Oct4, Sox2, Fgfr2 and Gli2 measured by qRT-PCR. All data are represented as mean ± SD; n=3 and p&lt;0.05 (one-way ANOVA and Bonferroni’s post-test).  </div></div>
 <p>&nbsp;</p>
</body>

</html>
