<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">


<head>
<link href="../../styles/style2.css" rel="stylesheet" type="text/css" />
<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
<title>Figure S1. Esrrb rescue clone and experimental time course controls for different regulatory layers. </title>
</head>

<body style="width: 1280px; height: 1459px;"/>

 <div class="image_popup" style="width: 1200px">
<img alt="Figure 1" src="sfigure1.png"/><br>
<div style="width: 1200px; height: 107px">
	<strong>Figure S1. Esrrb rescue clone and experimental time course controls for different regulatory layers. <br />
	</strong> 
	(A) Esrrb expression levels after Dox withdrawal. Endogenous levels of Esrrb mRNA are undetectable due to the constitutive shRNA (see Fig 1A). Quantitative PCR data confirm expression of the exogenous form in presence of Dox (mean ± SD for 3 replicates (n=3)). (B) Western blot confirms downregulation of Esrrb and GFP in the absence of Dox. (C) Flow cytometry analyses of SSEA1 cell surface marker levels upon depletion of Esrrb. (D) Specificity of the anti-Esrrb antibody used for ChIP. Enrichment of Esrrb-bound chromatin was confirmed by Western blots using IgG as a non-specific control. The arrow indicates Esrrb. (E) Differentiation of mESCs shown by gradual changes in morphology and loss of alkaline phosphatase (AP) activity in cells maintained for 5 days without Dox. In presence of Dox cells maintain an undifferentiated phenotype. (F) Distribution of Esrrb ChIP-seq tags relative to the TSS. Tag counts were normalized to the total number of tags in each sequencing reaction. (G) Gene Ontology (GO) analysis was performed to identify over-represented functional categories. Biological Process with significant over-representation are shown (p <0.01). Black bars represent the number of Esrrb-bound genes present in the respective GO category. White bars represent the number of genes expected by chance. (H) Subcellular localization controls for iTRAQ samples. Equal amounts of total protein from cytosolic and nuclear fractions were analyzed by Western blots using anti-Tubulin (cytosolic marker), anti-Calnexin (endoplasmic reticulum marker) and anti-histone-H3 (nuclear marker). Western blot confirmation of Esrrb depletion and nuclear localization over the depletion time-course. (I) Hierarchical clustering of the significant log2 fold protein level changes during the time-course. (J) Protein levels measured by iTRAQ correlate with Western blot results for Dnmt3a, Mov10, Klf5 and Creb1. 
  </div>
</div>
 <p>&nbsp;</p>
</body>

</html>
