<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">


<head>
<link href="../../styles/style2.css" rel="stylesheet" type="text/css" />
<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
<title>Figure S6 Esrrb regulated mRNA/miR feed-forward motifs.</title>
</head>

<body style="width: 1280px; height: 1762px;"/>

 <div class="image_popup" style="width: 1200px; height: 1760px;">
<img alt="sFigure 6" src="sfigure6.png"/><br>
<div style="width: 1200px; height: 147px">
	<strong>Short RNA analysis and annotation pipeline for miR sequencing data. In relation to Figure 3.
	<br />
	</strong>(A) Simple representation of the steps applied to analyze short RNA (sRNA) sequence data shown using <a href="http://cancan.cshl.edu/publicgalaxy/">http://cancan.cshl.edu/publicgalaxy/</a> (B) Stacked-histogram graph showing the nucleotide sequence distribution in the Solexa library. (C) Pipeline applied to sRNA sequence annotation. (D) Genomic distribution of sRNAs at different days from Fastq files. Pie charts represent the distribution of the eight different classes of sRNAs measured. ‘Rest small RNA’ represents UCSC database annotated sRNAs filtered from mitochondria, the Y-chromosome, repeat sequences, mRNAs, miRs, snRNAs and snoRNAs. ‘Unannotated’ represents sRNAs with no information in the UCSC database. The total numbers of reads were mapped to the mouse reference genome (July 2007, NCBI37/mm9). (E) Hierarchical clustering of the most significant changes in miR levels after depletion of Esrrb measured by microarrays (Exiqon). (F) Hierarchical clustering of the log2 fold changes measured by miR-seq.
	 </div>
</div>
	<strong>
 <p>&nbsp;</p>
</body>

</html>
